uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/repositories/DistributionManagementArtifactRepository.java

```diff
@@ -17,20 +17,11 @@
 
 package org.guvnor.m2repo.backend.server.repositories;
 
-import java.io.ByteArrayInputStream;
```

Comment:
I think you are not using the proper [code style](https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/main/ide-configuration)

Reply:
yes you are right, I had not included the kie codestyle there. Its done now. Thanks